addons/mail/static/src/components/file_uploader/file_uploader.js

```diff
@@ -25,8 +15,7 @@ export class FileUploader extends Component {
     setup() {
         super.setup();
         useComponentToModel({ fieldName: 'component', modelName: 'FileUploaderView', propNameAsRecordLocalId: 'localId' });
```

Comment:
I think you removed all occurrences of `.component` with this PR? So this and the corresponding field could be removed.

Reply:
Comment still open here, can't this component ref be removed?